NonBayesian/First/first.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+from numpy.random import choice as generator
+import matplotlib.pyplot as plt
+
+def random_histogram(n):
+    res = np.random.rand(n)
+    return res / np.sum(res)
+
+
+# def generator(probability, length, k):
+# return np.random.choice(k, length, p=probability)
+
+
+def square_penalty(k, weight, j):
+    """
+        >>> [square_penalty([10, 20, 30, 40, 50],[0.1, 0.4, 0.2, 0.1, 0.2], i) for  i in range(0,5)]
```

Comment:
You can split the line and should not specify `0` as the initial element of the `range`. Also, why do you have such a big indent before?
For example this way
```suggestion
    >>> [square_penalty([10, 20, 30, 40, 50], [0.1, 0.4, 0.2, 0.1, 0.2], i)
    ...  for i in range(5)]
```

Also, the test is too complicated. Isn't it enough to have only two values?

The same thing is further. Won't comment duplicated issues.